src/auth/UserRequest.cc

```diff
@@ -172,11 +185,11 @@ authenticateAuthUserRequestIPCount(Auth::UserRequest::Pointer auth_user_request)
 /*
  * authenticateUserAuthenticated: is this auth_user structure logged in ?
  */
-int
+bool
 authenticateUserAuthenticated(Auth::UserRequest::Pointer auth_user_request)
```

Comment:
```suggestion
authenticateUserAuthenticated(const Auth::UserRequest::Pointer &auth_user_request)
```

Reply:
Hm.. const reference sounds like a good idea. Const by itself was removed as unnecessary

Reply:
Yes, our smart `Pointer` types should always default to being const-reference parameters to avoid complex copying and locking actions.